tests/unit/test_processing.py

```diff
@@ -507,7 +509,14 @@ def test_tensorflow_processor_with_required_parameters(
     else:
         tensorflow_image_uri = (
             "763104351884.dkr.ecr.us-west-2.amazonaws.com/tensorflow-training:{}-cpu-{}"
-        ).format(tensorflow_training_version, tensorflow_training_py_version)
+        ).format(
+            (
+                "2.16"
```

Comment:
Will change this condition to extract the `major.minor` version from the FW so that it can be used for future-releases if needed.